server/src/main/java/org/candlepin/liquibase/LiquibaseCustomTaskWrapper.java

```diff
@@ -69,6 +69,7 @@ public void execute(Database database) throws CustomChangeException {
                 .newInstance(database, new LiquibaseCustomTaskLogger());
 
             task.execute();
+            task.close();
```

Comment:
Maybe put this in a `finally` block? Or you could use the new `try-with-resources` syntax.